setup.py

```diff
@@ -14,8 +19,9 @@
     author_email='info@scrapinghub.com',
     url='http://github.com/scrapinghub/python-scrapinghub',
     platforms = ['Any'],
-    py_modules = ['scrapinghub'],
-    install_requires = ['requests'],
+    py_modules = ['scrapinghub', 'hubstorage'],
```

Comment:
Updated structure & fixed imports: now it fits the plan. Last thing is proper versioning - I guess we're going to have a single version for the merged repository. 
Should we do it in the same PR or update it later? I'd use Scrapy approach: 
- `VERSION` file in `scrapinghub/`
- `setup.py` / `scrapinghub.__init__` reading from `VERSION`